gulpfile.js

```diff
@@ -46,4 +49,14 @@ gulp.task('test', function(done) {
   }, done);
 });
 
-gulp.task('default', ['lint', 'test']);
+gulp.task('minify', function(done) {
+  return gulp.src(__dirname + '/src/**/*.js')
```

Comment:
File order is important here (look at karma configuration). I think you should probably put an array of file as parameter ?